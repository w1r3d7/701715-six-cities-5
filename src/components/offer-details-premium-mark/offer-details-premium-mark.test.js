import React from 'react';
import renderer from 'react-test-renderer';

import OfferDetailsPremiumMark from './offer-details-premium-mark';

describe(`OfferDetailsPremiumMark render correctly`, () => {
  it(`Is Premium`, () => {
    const tree = renderer
      .create(
          <OfferDetailsPremiumMark isPremium={true} />
      ).toJSON();

    expect(tree).toMatchSnapshot();
  });

  it(`Is not Premium`, () => {
    const tree = renderer
      .create(
          <OfferDetailsPremiumMark isPremium={false} />
      ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});

