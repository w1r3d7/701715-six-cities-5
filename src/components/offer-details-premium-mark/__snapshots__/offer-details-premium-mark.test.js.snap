// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OfferDetailsPremiumMark render correctly Is Premium 1`] = `
<div
  className="property__mark"
>
  <span>
    Premium
  </span>
</div>
`;

exports[`OfferDetailsPremiumMark render correctly Is not Premium 1`] = `null`;
